modules/ROOT/pages/mergetags.adoc

```diff
@@ -0,0 +1,68 @@
+= Merge Tags plugin
+:navtitle: Merge Tags
+:description: Enables Merge Tags functionality in the {productname} editor
+:keywords: merge, tags, tag, mergetags, mergetag, mergetags_list, mergetags_prefix, mergetags_suffix, mail merge, personalization token
+:pluginname: Merge Tags
+:plugincode: mergetags
+:pluginminimumplan: tiertwo
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows the user to insert a merge tag (also known as a personalization token, or a mail merge field).
+
+Merge tags can be inserted as-you-type; selected from a drop-down list when a specified prefix is typed; or selected and inserted from the searchable Merge tags toolbar menu button.
+
+Once a merge tag is inserted, the plugin leaves a non-editable variable wrapped with a prefix and suffix, making it easily identifiable.
+
+== Interactive example
+
+liveDemo::merge-tag[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'mergetags',
+  toolbar: 'mergetags',
```

Comment:
Should add `mergetags_list` option since the menu item is not shown without it

Reply:
This will have to include a basic list of mergetags then as well.

Reply:
Added `mergetags_list` option and ‘a basic list of mergetags’ to the basic setup example.